lcd.js

```diff
@@ -237,6 +254,12 @@ Lcd.prototype._send = function (val, mode) {
 
 // private
 Lcd.prototype._write4Bits = function (val) {
+  if(!(typeof val === 'number')){
+    throw new Error("Value passed to ._write4Bits must be a number");
+  } else if (val > 16){
+    throw new Error("Value passed to .write4Bits must be 16 or less")
```

Comment:
Currently it's valid to call _write4Bits with a val > 16 and this happens a lot. Because it happens a lot, the error on line 260 will be thrown a lot. For example, the command for SCROLL_LEFT is 0x18, so when scrollDisplayLeft is called, _write4Bits ends up being called twice from _send. In the first call val is 0x01 and in the second call it's 0x18. The error will be thrown in the second call. The same will happen when most characters are printed as most characters have a character code > 16